libs/shared/cms/src/lib/queries/relying-party/types.ts

```diff
@@ -23,6 +23,14 @@ export interface Shared {
   emailLogoAltText: string | null;
   backgroundColor: string | null;
   pageTitle: string | null;
+  headerLogoUrl: string | null;
+  headerLogoAltText: string | null;
+  featureFlags: FeatureFlags | null;
+}
+
+export interface FeatureFlags {
+  syncConfirmedPageHideCTA: boolean | null;
+  syncHidePromoAfterLogin: boolean | null;
```

Comment:
These two property cannot be null right?  Strapi doesn't allow an admin to add the feature flags while leaving those values as nulls.

Reply:
Yea, they should have a value. Typescript was complaining about mismatch values though.